backend/src/main/resources/db/migration/V3__add_email_send.sql

```diff
@@ -0,0 +1,2 @@
+alter table patients
+    add email_sent_date timestamptz;
```

Comment:
Chpau, ze tohle je jen pro overeni, ze se email odeslal?

Reply:
jo, presne tak

navic jak to bezi asynchrone, tak se muze teoreticky stat, ze kdyz nam umre server, tak nejake maily nebudou odeslane, proto je fajn to tam tohle mit